Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -297,6 +297,16 @@ def initializeParsedPreferences(parsedPreferences)
             end
         end
 
+        # Verify that nothing is enabled by default for experimental preference
+        if %w{ unstable testable preview }.include?(status)
+            options["defaultValue"].each do |component, values|
+              # FIXME: This doesn't check for more dynamic cases (when the value is a String containing runtime condition,...)
+              if (values["default"] == true)
+                reject.call "Preference #{name} is experimental (#{status}) but is activated by default for #{component}"
```

Comment:
```suggestion
                reject.call "Preference #{name} is #{status} but is activated by default for #{component}"
```

Can we just write this directly?

Reply:
Also activated has a different meaning in english, "enabled by default" might be better.

Reply:
Changed